composure.sh

```diff
@@ -329,9 +329,10 @@ glossary ()
   group 'composure'
 
   typeset targetgroup=${1:-}
-  typeset maxwidth=$(_longest_function_name_length | awk '{print $1 + 5}')
+  typeset functionlist="$(_typeset_functions)"
+  typeset maxwidth=$(_longest_function_name_length "$functionlist" | awk '{print $1 + 5}')
 
-  for func in $(_typeset_functions); do
+  for func in $functionlist; do
```

Comment:
Is there really no way to avoid calling _typeset_functions_about twice? A single call to this function lasts about 4 seconds for me using cygwin. So this is significant in this environment.

Using

    setopt shwordsplit

would allow

    for func in $functionlist; do

in zsh, but that would require another shell switch/case.

As _typeset_functions allready contains a shell switch/case: Would it be possible that the function returns a space-seperated list for bash, but an array for zsh? The following seems to work in zsh:

    funclist=(a b c)
    for func in $funclist; do